### DISCUSSION TITLE
Nested Menu with Aria?

### DISCUSSION THREAD

**Discussion by torgeadelin** [adobe/react-spectrum#2400] [Q&A] [answered]
Hi,

I've recently build  menu / menu-trigger components using aria and stately, and I'm wondering if it's possible to create nested menus using `useMenu` somehow? I find it tricky because of the restricting structure of how we provide the menu-items via the `Item` component inside the collection API from stately.

Here's an example of what I'm trying to achieve.

![image](https://user-images.githubusercontent.com/14194394/135428091-29332227-5bdc-4f0a-b1b1-6334fa49f620.png)

I have some ideas already but they seem hacky, and I want to know if there's a proper mechanism to do this before I try implementing.

Thanks!

**Comment by reidbarber** [comment, marked as answer]
Hi @torgeadelin, this isn't supported at the moment, but there is an existing issue for it at https://github.com/adobe/react-spectrum/issues/1356.
